### PR TITLE
Use multi-arch test images & few changes

### PR DESCRIPTION
**Pull request by pratham-m**
This PR intends to add multi-arch support for the groovy tests.
Test images used in respective test are changed to use alternate multi-arch images which support 3 platforms viz. `x86_64`, `ppc64le` and `s390x`.

Changes are verified manually using `./gradlew test --tests=TestName`

### CODE REVIEW

**Comment by pratham-m** [comment]
Hi @gavin-stackrox  @robbycochran can you kindly check the changes ?

File: qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy [hunk 1]
```diff
@@ -125,7 +125,7 @@ class DeploymentEventGraphQLTest extends BaseSpecification {
             log.info "return code " + depEvents.getCode()
             assert depEvents.getValue().result != null
             def events = depEvents.getValue().result
-            assert events.numPolicyViolations == 1
```
Comment (gavin-stackrox) [inline qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy:128]: IIUC this test expects only "Ubuntu Package Manager Execution" to be triggered. But under P/Z more than that is triggered. That works for other cluster flavors, so I would prefer this to be a:
```
if P/Z
  >= 1
else
  == 1
```
Even better would be to explain what other policies are triggered and why this needs to be >= 1 on P/Z.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

File: qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy (hunk 1 above)
Comment (pratham-m) [reply qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy:128]: ACK - will make the changes

**Review by gavin-stackrox — APPROVED** [review]
Looks ready to merge. Thanks @pratham-m

**Comment by pratham-m** [comment]
> Looks ready to merge. Thanks @pratham-m

Thanks @gavin-stackrox

**Pull request merged**